config/application.yml.default

```diff
@@ -14,17 +14,17 @@
 
 # Make sure any new entries you add are enclosed in single quotes.
 
-aamva_auth_request_timeout: '5'
+aamva_auth_request_timeout: 5.0
 aamva_auth_url: 'https://example.org:12345/auth/url'
-aamva_cert_enabled: 'true'
+aamva_cert_enabled: true
 aamva_sp_banlist_issuers: '[]'
-aamva_verification_request_timeout: '5'
+aamva_verification_request_timeout: 5.0
 aamva_verification_url: https://example.org:12345/verification/url
-all_redirect_uris_cache_duration_minutes: '2'
-account_reset_token_valid_for_days: '1'
-account_reset_wait_period_days: '1'
-acuant_maintenance_window_start:
-acuant_maintenance_window_finish:
+all_redirect_uris_cache_duration_minutes: 2
+account_reset_token_valid_for_days: 1
+account_reset_wait_period_days: 1
+acuant_maintenance_window_start: 
+acuant_maintenance_window_finish: 
```

Comment:
looks like with a blank value we're adding trailing spaces, let's avoid this if we can:
```suggestion
acuant_maintenance_window_start:
acuant_maintenance_window_finish:
```